hack/e2e.sh

```diff
@@ -15,6 +15,14 @@ function cleanup() {
         echo "Removing namespaces with the 'name=${METERING_NAMESPACE}-metering-testing-ns' label"
         kubectl delete ns -l "name=${METERING_NAMESPACE}-metering-testing-ns" --wait=false || true
 
+        # Remove any testing labels that may have been added during tests
+        echo "Removing any testing labels that were added to the cluster's nodes"
+        nodes=( $(kubectl get nodes -l metering-node-testing-label="true" --no-headers | awk '{ print $1 }') )
+        for i in "${nodes[@]}"
+        do
+            kubectl label node "$i" metering-node-testing-label- 2>/dev/null
+        done
```

Comment:
This was added to the cleanup function that's run anytime the EXIT pseudo-signal has been encountered. Probably need to tinker with this more further down the line, as we're not always going to be running this node selector configuration test, but it should be sufficient for now.